### ISSUE TITLE
Weird redirect when adding page with alternate User template

### ISSUE THREAD

**Issue by thetuningspoon** [ryancramerdesign/ProcessWire#1840]
I am taking advantage of the new ability to create users of a template different from "user" and adding them in a different part of the page tree. But when I go to create a new page from the page tree using the alternate user template, some kind of weird redirect happens and I end up on /access/users/edit/bookmarks/bookmarks/bookmarks/bookmarks/

If I remove the alternate "Page Class Name" setting ("User") then I can add new pages again, but the users I create cannot login.

PW 3.0.16

**Comment by ryancramerdesign** [comment]
Does it work if you add them from the Access > Users instead? I'm pretty sure we've blocked the ability to edit users outside of ProcessUser. Meaning, if you try to edit a user from the page tree, it should redirect to ProcessUser. Seems like that redirect isn't working in the case you've mentioned though.

**Comment by thetuningspoon** [comment]
Yes, it does work when adding from Access > Users.

**Comment by thetuningspoon** [comment]
Ryan, I'm taking a closer look at this since this project will be getting passed off to the client soon. any suggestions off the top of your head on where to look for the redirect that's getting mixed up?

**Comment by thetuningspoon** [comment]
Hi Ryan, I spent some time in the code and I think the cause of this was that I was trying to use the quick one-step page creation (auto-naming) feature for these user pages. It may also have been due to the ProcessSetupPageName module. I switched back to the 2-step page creation where it asks for the username up front, and all is well again. This is not ideal since I didn't want the user to have to think about the username or password unless they really wanted the user to be able to login, but it's ok for now.
